client/src/components/Tabs/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import SwipeableViews from 'react-swipeable-views';
+import { withStyles } from '@material-ui/core/styles';
+import { AppBar, Tabs, Tab, Box, Typography } from '@material-ui/core';
+import styles from './style';
+import AcoountsInfo from '../AccountInfo';
+import ScoresInfo from '../ScoresInfo';
+import axios from 'axios';
+import { ValueConsmer } from '../../screens/PersonalProfile/contextProvider';
+
+function TabPanel(props) {
+	const { children, value, index, ...other } = props;
+
+	return (
+		<div
+			role="tabpanel"
+			hidden={value !== index}
+			id={`full-width-tabpanel-${index}`}
+			aria-labelledby={`full-width-tab-${index}`}
+			{...other}
+		>
+			{value === index && (
+				<Box p={3}>
+					<Typography component={'span'}>{children}</Typography>
+				</Box>
+			)}
+		</div>
+	);
+}
+
+TabPanel.propTypes = {
+	children: PropTypes.node,
+	index: PropTypes.any.isRequired,
+	value: PropTypes.any.isRequired,
+};
+
+function a11yProps(index) {
+	return {
+		id: `full-width-tab-${index}`,
+		'aria-controls': `full-width-tabpanel-${index}`,
+	};
+}
+class TabsComponent extends Component {
+	state = {
+		userId: 1,
+		data: {},
+	};
+	componentDidMount() {
+		axios
+			.post('/api/user/:userId/profile', this.state)
```

Comment:
you should use get instead of post, here you want get personal information from the db